app/containers/DeleteAccount/index.js

```diff
@@ -67,13 +75,22 @@ export class DeleteAccount extends React.PureComponent {
           {content === SUBMIT_EMAIL_FORM && (
             <SubmitEmailForm
               locale={locale}
-              deleteAccount={deleteAccountDispatch}
+              deleteAccount={
+                loginWithFacebook
+                  ? deleteFacebookAccDispatch
+                  : deleteAccountDispatch
+              }
               deleteAccountProcessing={deleteAccountProcessing}
+              loginWithFacebook={loginWithFacebook}
             />
           )}
         </Modal>
 
-        {render({ onClick: showDeleteAccountModalDispatch })}
+        {render({
```

Comment:
I think we should show modal anyway. Even if it is facebook.
Do we show any message to the user that email was sent?

Reply:
The whole deletion flow is saved. I used sendEmailFacebookDispatch just to pass facebook user email to sendEmailWorker saga. I had to use sendEmailFacebook action to reconcile data coming from email deletion form and facebook user data.

Reply:
The whole deletion flow is saved. I used sendEmailFacebookDispatch just to pass facebook user email to sendEmailWorker saga. I had to use sendEmailFacebook action to reconcile data coming from email deletion form and facebook user data.